include/aikido/distance/ConfigurationRanker.hpp

```diff
@@ -0,0 +1,56 @@
+#ifndef AIKIDO_DISTANCE_CONFIGURATIONRANKER_HPP_
+#define AIKIDO_DISTANCE_CONFIGURATIONRANKER_HPP_
+
+#include "aikido/distance/CartesianProductWeighted.hpp"
+#include "aikido/distance/DistanceMetric.hpp"
+#include "aikido/distance/defaults.hpp"
+#include "aikido/statespace/CartesianProduct.hpp"
+#include "aikido/statespace/dart/MetaSkeletonStateSpace.hpp"
+
+#include <dart/dynamics/dynamics.hpp>
+
+namespace aikido {
+namespace distance {
+
+class ConfigurationRanker
+{
+public:
+  /// Constructor
+  ///
+  /// \param[in] metaSkeletonStateSpace Statespace of the skeleton.
+  /// \param[in] metaskeleton Metaskeleton of the robot.
+  ConfigurationRanker(
+      statespace::dart::ConstMetaSkeletonStateSpacePtr metaSkeletonStateSpace,
+      ::dart::dynamics::ConstMetaSkeletonPtr metaSkeleton);
+
+  /// Destructor
+  virtual ~ConfigurationRanker() = default;
+
+  /// Returns the statespace.
+  statespace::ConstStateSpacePtr getStateSpace() const;
```

Comment:
This method doesn't seem to be used. I'd prefer to remove it if we don't see a use for it, but I think this should return a `ConstMetaSkeletonStateSpacePtr` if we decide to keep it.

Reply:
I'll remove it for now.